wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java

```diff
@@ -1162,6 +1174,30 @@ private void throwWsWUExceptions(ArrayOfEspException wsWUResponseExceptions, Str
         throw new Exception(multimessage.toString());
     }
 
+    /**
+     * Creates and throws exception with exception message response from WS
+     *
+     * @param eclExceptions - the array of ECLException objects to throw
+     * @param message - the prefix message
+     * @throws Exception
+     */
+    private void throwWsWUExceptions(ECLException[] eclExceptions, String message) throws Exception
```

Comment:
Let's distinguish this method from the existing throwWsWUExceptions by labeling it throwWUECLExceptions